pkg/accounting/accounting.go

```diff
@@ -595,6 +597,60 @@ func (a *Accounting) Balances() (map[string]*big.Int, error) {
 	return s, nil
 }
 
+type PeerInfo struct {
+	Peer                  string
```

Comment:
there's no need to represent the peer address twice. since the endpoint serializes a map with address as key, you're repeating the data here